### DISCUSSION TITLE
SSL / https and Multi-Domain (2nd domain on the same folder/path)

### DISCUSSION THREAD

**Discussion by AWqxKAWERbXo** [leokhoa/laragon#918] [Q&A]
Hey guys,

I set up my CMS to work as multi-domain from the same project folder, e.g. `C:\laragon\www\example.com`, several domains point to the folder, e.g. `example.com.test`, `example.org.test`, `example.net.test`.

1. I manually added the domains to the Windows Hosts-File.
2. I manually duplicated and modified the `.conf` file ("sites enabled") for apache

The websites work as expected, which means, they are functional in my local browser excepted for https.

Laragon "magically" enables SSL for them - which steps do I need to perform to make the websites available via https?

**Comment by Pen-y-Fan** [comment]
Possibly these discussions may help: 

- Configuring subdomain in Laragon #584 
- force http to https during installation of wordpress #409
